src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -350,27 +368,36 @@ def read_forward_model(
                         for key, val in args:
                             job.private_args.addItem(key, val)
                 except ValueError as err:
-                    raise ConfigValidationError(
-                        errors=f"{err}: 'FORWARD_MODEL {job_name}({args})'\n",
-                        config_file=config_file,
-                    ) from err
+                    all_errors.append(
+                        ConfigValidationError(
+                            errors=f"{err}: 'FORWARD_MODEL {job_name}({args})'\n",
+                            config_file=config_file,
+                        )
+                    )
```

Comment:
Need to use `continue` after the except.

Reply:
Done, added continue after missing fwd models / sim jobs